index.html

```diff
@@ -58,5 +64,142 @@ <h1>Hey there. I’m <br>Lukman
         </div>
         </div>
     </section>
+
+    <!--Midde Section-->
+
+    <section>
+        <main>
+            <div class="container-works">
+                <h2 class="subtitle">My Recent<br> <span class="underline">Works</span></h2>
+                <hr>
+            <div class="projectlist">
+                <div class="project1">
+                    <div class="image">
+                        <img class="projectimage" src="/images/project-img.png" alt="project desc"></div>
+                    <div class="projectdetail">
+                        <h2 class="projectname">Project 1</h2>
+                    <ul class="skills">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+                    <button type="button" class="projectbutton">See Project</button>
+                </div></div>
+                <div class="project2"></div>
+            </div>
+            <div class="projectlist">
+                <div class="project1">
+                    <div class="image">
+                        <img class="projectimage" src="/images/project-img.png" alt="project desc"></div>
+                    <div class="projectdetail">
+                        <h2 class="projectname">Project 1</h2>
+                    <ul class="skills">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+                    <button type="button" class="projectbutton">See Project</button>
+                </div></div>
+                <div class="project2"></div>
+            </div>
+            <div class="projectlist">
+                <div class="project1">
+                    <div class="image">
+                        <img class="projectimage" src="/images/project-img.png" alt="project desc"></div>
+                    <div class="projectdetail">
+                        <h2 class="projectname">Project 1</h2>
+                    <ul class="skills">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+                    <button type="button" class="projectbutton">See Project</button>
+                </div></div>
+                <div class="project2"></div>
+            </div>    
+            
+        </div>
+        </main>
+    </section>
+    <section>
+        <div class="container-about">
+            <div class="aboutimage"><h2>About me <img class="img-foot" src="/images/illustration 1- about me-mobile.svg" alt=""> </h2></div>
```

Comment:
Kindly note that your background in the about section does not match the Figma design. Also, Instead of putting the image in the heading of the about section, please use it as a background image 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/192093676-c51cb706-d00b-4aae-8075-9b365e32d50d.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/192093687-6a7c76ee-7717-4ab1-8e5d-1ddfaf339563.png)